app/src/main/java/net/squanchy/favorites/view/FavoritesItem.kt

```diff
@@ -0,0 +1,34 @@
+package net.squanchy.favorites.view
+
+import net.squanchy.schedule.domain.view.Event
+import org.joda.time.LocalDate
+
+sealed class FavoritesItem {
+
+    abstract val id: Long
+
+    abstract val type: Type
+
+    data class Header(val date: LocalDate) : FavoritesItem() {
+
+        override val id: Long
+            get() = date.toDateTimeAtCurrentTime().millis // We're relatively sure this won't clash with event IDs
```

Comment:
The alternative would be to get the `dayId` but that has pretty much the same disadvantage of risking to clash with `eventId`s, and an extremely high likeliness of clashes if the various IDs are numeric to begin with.

Note to future self: use GUIDs for everything in the database.